### DISCUSSION TITLE
'Back to Home' button is not working when installed in a sub directory

### DISCUSSION THREAD

**Discussion by nikitakothari06** [LycheeOrg/Lychee#2635] [Bugs] [answered]
### Lychee version

6.0.1

### Which PHP version are you using?

PHP 8.2

### Detailed description of the problem

Hi,

I am trying to install Lychee in a subdirectory and it is working fine after resolving #2603

However when I click on the 'cancel' button on login page it shows the option on top right corner to return to home. When I click on "Return to Home" it redirects to the root directory.

![image](https://github.com/user-attachments/assets/ab9a7e37-4f75-4006-9f0d-4b1a2fd00b40)

I have installed Lychee in a subdirectory with PHP 8.2, Mysql 5.7 and following settings in .env file
APP_URL=https://[[domain.com]]
APP_DIR=/[[sub-dir]]/public

### Steps to reproduce the issue

1. Go to login page.
2. Press the cancel button.
3. Click on "Return to Home" button.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by nikitakothari06** [comment]
One more issue I encountered is I am unable to add anything on "On this Day" albaum. 

It shows that the image is uploaded successfully but on the homepage it shows no photos in this albaum.

![image](https://github.com/user-attachments/assets/60f66911-15ac-4f95-b911-7f5afe66aa4f)

**Reply by ildyria** [reply]
On this day album takes the taken_at field. I'm not sure if it takes the created at one. Will need to check.

**Reply by d7415** [reply]
> One more issue I encountered is I am unable to add anything on "On this Day" albaum.

Just to confirm, are you trying to upload a photo with dates set to today, or are you trying to upload to the On This Day smart album?

**Comment by ildyria** [comment, marked as answer]
So after further investigation, I am going to flag this one as Not an issue.
The reason is that the user is supposed to configure that button in his settings.
Left menu => Settings => All Settings => Mod Back Button => Link of the back button

Why is the user supposed to configure it instead of using the `.env` values is because some people want the back home to be going back to their personal website for example and not necessarily to the landing page.

**Reply by ildyria** [reply]
What I will do however is add a warning that this link may need to be modified if you are hosting on a sub folder.
